### PR TITLE
Fix session TokenType

### PR DESCRIPTION
**Pull request by peetrike**
- make it same as used in TssSession class

### CODE REVIEW

**Comment by wsmelton** [comment]
Please include the changelog update as mentioned in contributing guide.

File: tests/functions/New-TssSession.Tests.ps1 [hunk 1]
```diff
@@ -98,5 +98,8 @@ Describe "$commandName works" {
             $currentTime = [datetime]::UtcNow
             $sessionAccessToken.StartTime | Should -BeLessOrEqual $currentTime
         }
+        It "Sets correct TokenType" {
```
Comment (wsmelton) [inline tests/functions/New-TssSession.Tests.ps1:101]: Please add this for the Credential context as well for consistency.

**Comment by peetrike** [comment]
will do

File: tests/functions/New-TssSession.Tests.ps1 (hunk 1 above)
Comment (peetrike) [reply tests/functions/New-TssSession.Tests.ps1:105]: in https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/282723876f9f3ca37e8e759f82c408b676e0567e

**Comment by wsmelton** [comment]
Right now OAuth2 token and Windows Auth are supported. The WinAuth endpoints are found [here](https://updates.thycotic.net/secretserver/restapiguide/10.9.33/WinAuth/)

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * None
 
+### Fixed
+
+* `New-TssSession` - fixed token type when AccessToken parameter is used. [26720ba](https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/26720ba9d37b3f2c6c8986a2f1b2ed30dc5c20e7)
```
Comment (wsmelton) [inline CHANGELOG.md:20]: This can be put under the changed section.

Comment (peetrike) [reply CHANGELOG.md:16]: in https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/f8082dc462a413652de34ca53e88c8cc9f2b0d4e

**Review by wsmelton — APPROVED** [review]

**Review by wsmelton — APPROVED** [review]

**Pull request merged**